xapian-applications/omega/daflist.h

```diff
@@ -0,0 +1,8 @@
+/** @file daflist.h
+ * @brief 'Extensions' of directories to be indexed as files.
```

Comment:
> `Extensions`

Why the quotes?

Reply:
I always thought _extension_ is meant for files only. Though directories are files in linux.

Reply:
There's not really a formal standard definition of "extension", but I think it's reasonable to think of it as part of a pathname whatever that pathname refers to.

E.g. https://en.wikipedia.org/wiki/Bundle_(macOS) says:
> Application bundles are directory hierarchies, with the top-level directory having a name that ends with a `.app` extension.

The use of extensions on the names of directories seems to be a mac thing - at least I can't think of examples that originate on other platforms.  That's certainly where `.key`, etc originate.